source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx

```diff
@@ -156,8 +143,16 @@ class SettingsRoute extends React.Component<IProps> {
 }
 
 export default compose(
-  query(mapRecordsToProps),
+  query(({match: { params: { orgId } } }) => ({
+    cacheKey: [`org-${orgId}`],
+    organization: [q => buildFindRecord(q, TYPE_NAME, orgId), buildOptions({
+      include: ['organization-product-definitions.product-definition', 'groups']
+    })],
+  })),
   withLoader(({ organization }) => !organization),
+  withOrbit(({ organization }) => ({
+    organization: q => q.findRecord({ type: TYPE_NAME,id: organization.id,})
```

Comment:
if findRecord takes an object, you could just do `q.findRecord(organization)` to keep things concise?

Reply:
also, @giancorzo is this the only place that needs this kind of treatment? or would it be worth it to roll this withOrbit generically into query?